frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx

```diff
@@ -129,20 +130,51 @@ export const AlertsSetup: React.FC = function () {
 		}
 	}, [alertsSetup, hidden])
 
+	const header = (
```

Comment:
You can probably define this as a file-wide constant at the top of the file.

Reply:
was feeling lazy here, good call! moved it into a component